app/views/document_exports/_modal.haml

```diff
@@ -1,4 +1,8 @@
-- data = { url: document_exports_path, status: 'pending' }
+- if request.path.match(/\/health\//)
+  - data_url = health_document_exports_path
+- else
+  - data_url = document_exports_path
+- data = { url: data_url, status: 'pending' }
```

Comment:
Is there an easy way to get the path out of the model?  This seems a bit brittle.

Reply:
@awisema, I think this is the one thing we should cleanup.